### PR TITLE
Review of the documentation for the chips part

### LINKED ISSUE #299: [Documentation] Missing documentation for chip_mdi*
**Describe the bug**
The template string in [chip_mdi_icon_state](https://ui-lovelace-minimalist.github.io/UI/usage/chips/chip_mdi_icon_state/#usage) and [chip_mdi_icon_only](https://ui-lovelace-minimalist.github.io/UI/usage/chips/chip_mdi_icon_only/#usage) is wrong.
It's `example` but should be `chip_mdi_icon_state` and `chip_mdi_icon_only`.

### PR DESCRIPTION
**Pull request by aehru**
Related to #294 and also fix #299 & #300

### CODE REVIEW

**Comment by CM000n** [comment]
Hi @aehru, thanks for your revision and additions to the chips docs. Unfortunately, your merge request creates conflicts to a large extent, since @sildehoop has already made revisions to the chips in the meantime. Could you please take another look at it.

File: docs/usage/chips/chip_presence_detection.md [hunk 1]
```diff
@@ -24,8 +24,8 @@ This `chip` shows you the actual presence in your home. Shows residents and gues
 - type: 'custom:button-card'
   template: chip_presence_detection
   variables:
-    ulm_chip_presence_counter_residents: sensor.my_residents_counter
-    ulm_chip_presence_counter_guests: sensor.my_guests_counter
+    ulm_chip_presence_counter_residents: sensor.people_home
+    ulm_chip_presence_counter_guests: sensor.people_home
```
Comment (CM000n) [inline docs/usage/chips/chip_presence_detection.md:28]: That should probably be rather `guests_home`? But as a example it is also ok, I think ;-)

Comment (aehru) [reply docs/usage/chips/chip_presence_detection.md:28]: Oh yes my bad. You could change it or you want me to do another push ?

Comment (CM000n) [reply docs/usage/chips/chip_presence_detection.md:28]: When you can do another push it would be great :-)

**Pull request merged**